libutils/mustache.c

```diff
@@ -252,6 +252,7 @@ static Mustache NextTag(const char *input,
         break;
     case '{':
         extra_end = "}";
+        break;
```

Comment:
So, here you've changed behavior, you changed something which was a fall through, to a break. Why did you make this change? There is no explaining comment, or explanation in the commit message.

Reply:
This is a mis-reading of the code on my part it seems. Seeing the rest of the fall throughs had a comment explaining that they are intentional i, apparently wrongly, assumed it was _not_ intentional and the break was missing.
Looking back at it i could have inferred that it was (intentional) because there is a pattern of breaking after setting up the `ret`. It should be changed to `/* fall through */`

Reply:
Makes sense. Next time, double check that you understand what the code is doing when making a change like this, and there should be an explanation as well.